packages/neon/neon/lib/src/widgets/error.dart

```diff
@@ -78,56 +95,65 @@ class NeonError extends StatelessWidget {
       color: color,
     );
 
-    final message =
+    final message = Text(
+      details.getText(context),
```

Comment:
Since the text is used multiple times it should be put into a variable to avoid recomputation.